scripts/publish-storybook-temp.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
Is this `-temp` file still needed? I don't see it referenced in .bacon.yml or the other scripts.

Reply:
Yes. I might switch over to this `-temp` method of hitting Aperture after I finish up the other tasks, so we're not having to hit S3 directly.